### PR TITLE
Thegraph artist component

### PR DESCRIPTION
**Pull request by andygray**
### New Changes

* [x] Artist Overview
* [x] Boards > Total Sales
* [x] body color change with @Selfmadeheroes 
* [x] tabs and button change with @Selfmadeheroes 

image
image
image

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -112,6 +112,20 @@ Vue.filter('dotDotDot', function (value) {
   return str;
 });
 
+// Apollo gql magic
+import ApolloClient from "apollo-boost";
+import VueApollo from "vue-apollo";
+
+const client = new ApolloClient({
+  uri: "https://api.thegraph.com/subgraphs/name/knownorigin/known-origin",
```
Comment (jamesmorgan) [inline src/main.js]: lets get this added to the readme as well @andygray  - also once this it out lets bang a few tweets to them so a few mentions and follows @Selfmadeheroes

File: src/components/pages/Boards.vue [hunk 2]
```diff
@@ -0,0 +1,79 @@
+<template>
+
+  <div class="mt-4 editions-wrap">
+
+    <b-tabs>
+
+      <b-tab title="Total sales" active>
+        <div class="mx-sm-5">
+          <table v-if="mostEth" class="table table-borderless">
```
Comment (jamesmorgan) [inline src/components/pages/Boards.vue]: If probs be able to make this sortable using vue-bootstrap https://bootstrap-vue.js.org/docs/components/table/#fields-as-an-array-of-objects 

Or another vue lib

File: src/components/pages/Boards.vue [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+            <tr>
+              <th colspan="2"></th>
+              <th>Editions</th>
+              <th>Sold</th>
+              <th>Total ETH</th>
+            </tr>
+          </thead>
+          <tbody>
+          <tr v-for="artist in mostEth">
+            <td class="text-left"><artist-image :logo="findArtistsForAddress(artist.address).logo" style="height: 50px; width: 50px" v-if="findArtistsForAddress(artist.address)"></artist-image></td>
```
Comment (jamesmorgan) [inline src/components/pages/Boards.vue]: Make this clickable to their profile

File: src/components/ui-controls/artist/ArtistPanel.vue [hunk 4]
```diff
@@ -14,24 +14,54 @@
...
       <p class="website">
         <a :href="artist.website" target="_blank">{{artist.website}}</a>
       </p>
     </div>
+
+    <div v-if="artistStats" class="mt-4 bio">
+      <hr/>
+      <h6>Artist overview</h6>
+
+      Total ETH <span class="ml-1">{{ artistStats.totalValueInEth }}</span><br/>
```
Comment (jamesmorgan) [inline src/components/ui-controls/artist/ArtistPanel.vue]: I thought the copy would be more like the frontpage e.g. 
Screenshot 2020-01-02 17 25 14

File: src/components/pages/Boards.vue (hunk 2 above)
Comment (andygray) [reply src/components/pages/Boards.vue]: You can do it but needs some tested and rework - I see the table not a something to sort - just a league table really. But one for discussion.

File: src/components/pages/Boards.vue (hunk 3 above)
Comment (andygray) [reply src/components/pages/Boards.vue]: Done

File: src/components/ui-controls/artist/ArtistPanel.vue (hunk 4 above)
Comment (andygray) [reply src/components/ui-controls/artist/ArtistPanel.vue]: I made the front page like this copy that @Selfmadeheroes approved and gave to me.

Comment (jamesmorgan) [reply src/components/ui-controls/artist/ArtistPanel.vue]: I dont mean the front page I mean the artist page - instead of a list have something like this

**Pull request merged**